update.sh

```diff
@@ -170,20 +195,21 @@ function main()
 
         mkdir $scenario_dir
 
-        prepare_session $scenario "$session_dir"/login $scenario_dir
+        prepare_session $scenario $session_dir/login $scenario_dir
 
+        source $endpoints_dir/.config
         # Loop through the endpoints, pump into output directory
-        for endpoint in "${endpoints[@]}"
+        for endpoint in "${ordered_endpoints[@]}"
         do
             echo "Preparing endpoint: $endpoint"
-            call_endpoint $scenario $endpoint $scenario_dir
+            call_endpoint $scenario $endpoints_dir/$endpoint $scenario_dir
         done
 
         # TODO: end session (delete)
     done
 
     # Remove the temp directory
-    # rm -rf $temp
+    rm -rf $temp
```

Comment:
Ouch. Wonder what could be in $temp. Worth prepending it with `/` to make sure there is no `~` or something absolute for example? I'd hate this script to execute something like `rm -rf ~` 🤔